### DISCUSSION TITLE
How does configuring work on Regolith 3.0?

### DISCUSSION THREAD

**Discussion by itsrllyhim** [regolith-linux/regolith-desktop#946] [Q&A]
I'm struggling to identify which file contains or needs to be edited, in order to do the following:

- I would like to change the default keybind to close applications and open the app launcher, as well as to view/manage notifications
- I would like to change the default browser

The documentation is mildly confusing for both of these, and my search in the discussions have not been helpful.

**Comment by gogsbread** [comment]
Regolith default configurations are overridden by values in `~/.config/regolith3/Xresources`. 
Say, you want to change the position of window manager bar to top, you can write a `wm.bar.position: top` in `Xresources`. grep `/usr/share/regolith` to find the default value of all the available kv pairs - `grep -R wm.bar.position /usr/share/regolith`
